doc/4-Merge.md

```diff
@@ -0,0 +1,349 @@
+- `def_class_missing_osm`: OSM contains an object, but no matching one found in the OpenData set. By convention the item number look like `7xxx` and class `id` is 2.
+- `def_class_possible_merge`: A probable matching object found. By convention the item number look like `8yy1` and class `id` is 3.
+- `def_class_update_official`: An OSM object matchs the OpenData one, close location and same reference value. But new tags or values from OpenData are available. By convention the item number look like `8yy2` and class `id` is 4.
+- `def_class_moved_official`: The matching OSM found, but not at the same location as OpenData. Not for main use case.
+
+All classes of issues are optional, but the main used one is `def_class_missing_official`.
+
+By convention the `level` value for this kind of issues is `3`, but other could be used.
+
+For other attributes, see common documentations.
+
+```python
+from modules.OsmoseTranslation import T_
+from .Analyser_Merge import Analyser_Merge
+
+class Analyser_Merge_Charging_station_FR(Analyser_Merge):
+    def __init__(self, config, logger=None):
+        self.def_class_missing_official(item = 8410, id = 1, level = 3, tags = ['merge'],
+            title = T_('Car charging station not integrated'))
+        self.def_class_missing_osm(item = 7051, id = 2, level = 3, tags = ['merge', 'post'],
+            title = T_('Car charging without tag "ref" or invalid'))
+        self.def_class_possible_merge(item = 8411, id = 3, level = 3, tags = ['merge'],
+            title = T_('Car charging station, integration suggestion'))
+        self.def_class_update_official(item = 8412, id = 4, level = 3, tags = ['merge'],
+            title = T_('Car charging station update'))
+```
+
+## OpenData set Source
+
+The OpenData set is available online and can be fetched by a URL. On edge case, the data can be included directly on the Osmose-QA backed repository, but we try to avoid this case.
+
+The OpenData set have this attributes:
+- URL of the Data, included in an archive or not:
+  - `fileUrl`: remote content file URL.
+  - `file`: local content file from the `merge_data` directory, avoid this case.
+- `attribution`: Author of the data, for the OSM `source` tag.
+- `millesime`: optional, date of the last release, for the OSM `source` tag. Note: since it is hard coded, it is not automatically updated when the remote data is updated. May be required with the attribution in some jurisdiction.
+
+Remote files are fetched and saved in the Osmose-QA Backend cache, the delay can be adjusted:
+- `fileUrlCache`: cache delay in days, default to 30 days.
+
+The remote file could be compressed or an archive:
+- `bz2` (boolean): the content is compressed in BZip2 format.
+- `gzip` (boolean): the content is compressed in GZip format.
+- `zip`: the remote URL is a ZIP archive and the data is at this path inside the archive.
+- `extract`: same as `zip`, but for all archive formats.
```

Comment:
"for all archive formats."   --> ça doit être quand même une archive ZIP ?

Reply:
No, any format, I added libarchive for that. But not used yet.